<launch>
	<include file="$(find match_gazebo)/launch/big_square.launch"/>
     
	<include file="$(find mir_launch_sim)/launch/mir_200.launch">
		<arg name="global_planner" value="GlobalPlanner"/>
    </include>
	
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find bachelor_rviz)/rviz/global_path.rviz"/>

	 <!-- Node 1: Gl채ttung durch B-Spline 3. Ordnung und Minimalen Radius -->
    <node name="kruemmung_node" pkg="bachelor_nodes" type="kruemmung.py" output="screen">
         <!-- Setze die Parameter f체r die Fahrzeugdynamik -->
    	<param name="robot_radius" type="double" value="0.5"/> <!-- Roboter-Radius in Metern -->
        <param name="max_steering_angle" type="double" value="0.785398"/> <!-- Maximaler Lenkwinkel in Radiant (45 Grad) -->
        <param name="smoothing_factor" type="int" value="10"/> <!-- s-Wert f체r B-Spline -->
    </node>

	<!-- Node 2: Pfade -->
    <node name="pfade_node" pkg="bachelor_nodes" type="pfade.py" output="screen">
    	<param name="num_alternative_paths" type="int" value="100"/> <!-- Anzahl der random Pfade-->
		<param name="smoothing_factor_pfade" type="int" value="10"/> <!-- s-Wert f체r B-Spline -->
	</node>
	
</launch>
